python/lsst/pipe/drivers/skyCorrection.py

```diff
@@ -113,14 +114,19 @@ def runDataRef(self, expRef):
         if DEBUG:
             extension = "-%(visit)d.fits" % expRef.dataId
 
+        if self.config.fakesAdded:
```

Comment:
This code block should not be duplicated twice, consider moving it to init and assigning to self, either with the name calexpType or something else if you are worried about that name being used elsewhere